pkgs/development/libraries/mesa/default.nix

```diff
@@ -73,6 +73,12 @@ stdenv.mkDerivation {
       })
     ];
 
+  postPatch = ''
+    substituteInPlace meson.build --replace \
```

Comment:
doesn't meson know of two pkg-config versions?

Reply:
Meson itself does, but for some reason this is using a generic find-program that doesn't know about the program it's looking for.